h2o-genmodel/src/main/java/hex/genmodel/tools/PrintMojo.java

```diff
@@ -1,32 +1,45 @@
 package hex.genmodel.tools;
 
+import guru.nidi.graphviz.engine.Format;
+import guru.nidi.graphviz.engine.Graphviz;
+import guru.nidi.graphviz.engine.GraphvizJdkEngine;
+import guru.nidi.graphviz.model.MutableGraph;
+import guru.nidi.graphviz.parse.Parser;
 import hex.genmodel.GenModel;
 import hex.genmodel.MojoModel;
 import hex.genmodel.algos.tree.SharedTreeGraph;
 import hex.genmodel.algos.tree.SharedTreeGraphConverter;
+import org.apache.log4j.BasicConfigurator;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 
-import java.io.File;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.PrintStream;
-import java.util.HashMap;
-import java.util.Map;
 
 /**
  * Print dot (graphviz) representation of one or more trees in a DRF or GBM model.
  */
 public class PrintMojo {
+
   private GenModel genModel;
-  private static boolean printRaw = false;
-  private static int treeToPrint = -1;
+  private boolean printRaw = false;
+  private int treeToPrint = -1;
   private static int maxLevelsToPrintPerEdge = 10;
-  private static boolean detail = false;
-  private static String outputFileName = null;
-  private static String optionalTitle = null;
-  private static PrintTreeOptions pTreeOptions;
-  private static boolean internal;
+  private boolean detail = false;
+  private String outputFileName = null;
+  private String optionalTitle = null;
+  private PrintTreeOptions pTreeOptions;
+  private boolean internal;
+  private String outputPngName = null;
+  private static final String tmpOutputFileName = "tmpOutputFileName.gv";
 
   public static void main(String[] args) {
+    // Provide configuration for graphviz logging
+    BasicConfigurator.configure();
+    // Select a way to process dot files
+    Graphviz.useEngine(new GraphvizJdkEngine());
```

Comment:
We could do this lazily, e.g. around line `250`. Only when the graph is actually printed to a file.

Reply:
Will fix